### PR TITLE
feat(HTML-report): Darkmode support

### PR DESCRIPTION
**Pull request by wijtserekker**
Updated the mutation-testing-elements dependency, and added a small script that changes the background of the page along with the theme of the report. This is located in the report-page component. It listens for the theme-changed event, and changes the background color of the parent of the mutation-testing-report-app accordingly.

### CODE REVIEW

**Comment by nicojs** [comment]
I've finally got the automated tests working again. 😅

I think it's not working 100% yet.

image

Did you run this locally? Let me know if you want help setting it up.

**Review by nicojs — CHANGES_REQUESTED** [review]
I've finally got the automated tests working again. 😅

I think it's not working 100% yet.

image

Did you run this locally? Let me know if you want help setting it up.

Demo: https://stryker-dashboard-acceptance.azurewebsites.net/reports/github.com/stryker-mutator-test-organization/hello-org/feat/modules

File: packages/website-frontend/src/app/report/report-page/report-page.component.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
 :host {
   --top-offset: 60px;
+  stryker-loading {
+    min-height: calc(100vh - 60px);
+  }
+  
+  .dark-report {
```
Comment (nicojs) [inline packages/website-frontend/src/app/report/report-page/report-page.component.scss:7]: This color will change https://github.com/stryker-mutator/mutation-testing-elements/pull/947. The background color will be available as `themeBackgroundColor`. 

So this is OK for now, and we'll have to update it once the new version is released.

File: packages/e2e/protractor.conf.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ const chromeOptions = {
   args: ['--window-size=800x600']
 };
 
-if (process.env.GITHUB_WORKFLOW) {
```
Comment (nicojs) [inline packages/e2e/protractor.conf.ts:8]: Why did you change this file? By mistake?

File: tasks/download-badge-api-package.sh [hunk 3]
```diff
@@ -2,7 +2,23 @@
 # After that, the package directory can be deployed to azure with package deploy
 set -e
 NPM_PACKAGE_VERSION=$1
-wget $(npm view @stryker-mutator/dashboard-badge-api@${NPM_PACKAGE_VERSION} dist.tarball)
+
```
Comment (nicojs) [inline tasks/download-badge-api-package.sh:5]: Did you change this file by mistake?

**Review by nicojs — CHANGES_REQUESTED** [review]
This looks good. Good solution IMO 👍

Some small remarks.

File: packages/website-frontend/src/app/report/report-page/report-page.component.scss (hunk 1 above)
Comment (wijtserekker) [reply packages/website-frontend/src/app/report/report-page/report-page.component.scss:7]: Good to keep in mind!

File: packages/e2e/protractor.conf.ts (hunk 2 above)
Comment (wijtserekker) [reply packages/e2e/protractor.conf.ts:8]: I believe you made these changes in 1efbd5bc6eddb139da133613eabbe0df96e12b20.

File: tasks/download-badge-api-package.sh (hunk 3 above)
Comment (wijtserekker) [reply tasks/download-badge-api-package.sh:5]: You made these changes in c95ad181d629f73c45af79dad198ce4e06f25fb3 :) My commits do not touch these files.

File: packages/website-frontend/src/app/report/report-page/report-page.component.scss (hunk 1 above)
Comment (hugo-vrijswijk) [reply packages/website-frontend/src/app/report/report-page/report-page.component.scss:7]: @wijtserekker 1.7.0 was just released with the new dark mode and `themeBackgroundColor` property

Comment (wijtserekker) [reply packages/website-frontend/src/app/report/report-page/report-page.component.scss:7]: Nice! I think I'll fix that version bump in another pull request. This one has been sitting here for way too long 😉.

File: packages/e2e/protractor.conf.ts (hunk 2 above)
Comment (nicojs) [reply packages/e2e/protractor.conf.ts:8]: ah wow! I didn't realize it wasn't merged back into master yet.

File: packages/website-frontend/src/app/report/report-page/report-page.component.scss (hunk 1 above)
Comment (nicojs) [reply packages/website-frontend/src/app/report/report-page/report-page.component.scss:7]: I'll do it since I'm fixing the workflow anyway

**Review by nicojs — APPROVED** [review]

**Comment by wijtserekker** [comment]
Whoo 🎊 , can't wait to see it live 😄

**Comment by nicojs** [comment]
image

😲

Hype!
